docs/pages/eas/workflows/examples/create-development-builds.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Create development builds
+description: Learn how to create development builds with EAS Workflows.
```

Comment:
```suggestion
description: Learn how to create development builds with EAS Workflows.
hideTOC: true
```

Please add `hideTOC` since all of these pages only have one section and showing a Table of content on the right side is unnecessary.